time stamp
uint16 curLeft
uint16 curRight
int32 posL
int32 posR
int16 gyroX
int16 gyroY
int16 gyroZ
int16 bemfL
int16 bemfR
int16 vBat
